.github/workflows/issue_triage.yml

```
@@ -1,11 +1,18 @@
-name: jit-triage
+name: Triage issues to Projects
 
 on:
   issues:
     types: [labeled]
 
 jobs:
-  welcome:
+  triage-issues:
+    strategy:
```

maybe add this just in case?
```suggestion
    strategy:
      fail-fast: false
```